lib/Features/Common/Home/View/home_page.dart

```diff
@@ -63,84 +64,91 @@ class _HomePageState extends State<HomePage> {
+                          ),
                         ),
                       ),
-                      child: Icon(Icons.camera_alt,
-                          color: _selectedSegment == 0
-                              ? Colors.white
-                              : Colors.black),
-                    ),
-                  ),
-                  GestureDetector(
-                    onTap: () {
-                      setState(() {
-                        _selectedSegment = 1;
-                      });
-                      Provider.of<CaptureProvider>(context, listen: false)
-                          .disposeCamera();
-                    },
-                    child: Container(
-                      width: 57,
-                      height: 32,
-                      decoration: BoxDecoration(
-                        color: _selectedSegment == 1
-                            ? Colors.blue
-                            : Colors.grey[300],
-                        borderRadius: const BorderRadius.only(
-                          topRight: Radius.circular(9),
-                          bottomRight: Radius.circular(9),
+                      GestureDetector(
+                        onTap: () {
+                          setState(() {
+                            _selectedSegment = 1;
+                          });
+                          Provider.of<CaptureProvider>(context, listen: false)
+                              .disposeCamera();
+                        },
+                        child: Container(
+                          width: 57,
+                          height: 32,
+                          decoration: BoxDecoration(
+                            color: _selectedSegment == 1
+                                ? ABColors.accent
+                                : Colors.white.withOpacity(0.25),
+                            borderRadius: const BorderRadius.only(
+                              topRight: Radius.circular(9),
+                              bottomRight: Radius.circular(9),
+                            ),
+                          ),
+                         child: Icon(
+                            Icons.chat,
+                            size: 16, // Set icon size to 16x16
+                            color: _selectedSegment == 1 ? Colors.black : Colors.white.withOpacity(0.5),
+                          ),
                         ),
                       ),
-                      child: Icon(Icons.chat,
-                          color: _selectedSegment == 1
-                              ? Colors.white
-                              : Colors.black),
-                    ),
+                    ],
                   ),
-                ],
-              ),
+                ),
+                //SWITCH
+                NotebookButton(
+                  onPressed: () {},
+                  iconColor: _selectedSegment == 0 ? Colors.white : Colors.white,
```

Comment:
langsung aja gausa pake ternary
```suggestion
                  iconColor: Colors.white,
```